__tests__/app.test.js

```diff
@@ -25,3 +24,29 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of all topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
```

Comment:
Good job getting the length check in here, the forEach would pass if the array was empty so the length check is important!